robot_smach_states/src/robot_smach_states/manipulation/grasp_point_determination.py

```diff
@@ -177,7 +185,3 @@ def visualize(self, candidates):
             #####
 
         self._marker_array_pub.publish(msg)
-
```

Comment:
Shouldnt there be one blank line at the end of file

Reply:
Yes there is. Otherwise github would have shown that.